chirp/drivers/rfinder.py

```diff
@@ -287,7 +287,7 @@ def __init__(self, *args, **kwargs):
 
         self._rfp = None
 
-    def set_params(self, (lat, lon), miles, email, password):
+    def set_params(self, lat, lon, miles, email, password):
```

Comment:
This does not work as expected. You've changed the signature from having 4 arguments (Not including `self`) to having 5 arguments. 

I don't see any updated callers reflect this change.